spec/lib/activitypub/tag_manager_spec.rb

```diff
@@ -7,17 +7,50 @@
 
   subject { described_class.instance }
 
+  let(:domain) { "#{Rails.configuration.x.use_https ? 'https' : 'http'}://#{Rails.configuration.x.web_domain}" }
```

Comment:
This pattern of checking this value to build an http-vs-https string is used in a few spots around specs (and maybe app?), could pull out some helper here as followup.